build_local.sh

```diff
@@ -57,6 +57,7 @@ compileSpecAndBuildLibrary(){
 buildACM(){
     # Build the ACM
     echo "${GREEN}Building ACM${NC}"
+    rm -r build
```

Comment:
Can we check if build folder exist before remove it?

Reply:
This check has been added to the script via https://github.com/CDOT-CV/asn1_codec/pull/11.